src/components/BulkOperationDetailsPage/BulkOperationDetails.test.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import {
+  render, screen, fireEvent, waitFor,
+} from '@testing-library/react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+
+import BulkOperationDetails from './BulkOperationDetails';
+
+const mockTask = {
+  task_id: '123',
+  task_type: 'course_create',
+  uploaded_by: 'test-user',
+  status: 'completed',
+  created: '2023-01-01T00:00:00Z',
+  modified: '2023-01-02T00:00:00Z',
+  csv_file: 'https://example.com/file.csv',
+  task_summary: {
+    errors: {},
+    others: [
+      'Course with key edX+CSL-700 already exists. Skipping creation',
+    ],
+    failure_count: 0,
+    success_count: 2,
+    created_products: [
+      '41413e7e-0e08-4e43-a795-87130bd08ac3 - Intro to Course Loader',
+      'b717187c-b8ae-47ac-b65b-ac2274e712db - Intro to Course Loader',
+    ],
+    total_products_count: 2,
+    updated_products_count: 0,
+  },
+};
+
+const csvContent = '"name","image"\n"Test Name","https://example.com/image.jpg"';
+
+describe('BulkOperationDetails', () => {
+  beforeEach(() => {
+    global.fetch = jest.fn().mockResolvedValue({
+      text: jest.fn().mockResolvedValue(csvContent),
+    });
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders task details correctly', () => {
+    render(
+      <IntlProvider locale="en">
+        <BulkOperationDetails task={mockTask} />
+      </IntlProvider>,
+    );
```

Comment:
Wrap this repeated render logic in a helper

Reply:
Moved rendering to `beforeEach` method